[clkmgr/rstmgr/doc] Add description of LPG output signals (#18105)
lowRISC/opentitan: hw/ip/rstmgr/doc/theory_of_operation.md

* [Interface Tables](../../../top_earlgrey/ip/rstmgr/data/autogen/rstmgr.hjson#interfaces)

> ## Reset Indications for Alert Handler

Reviewer: Move it after Reset Consistency Checks and make it ### ?

Author: done

> The alert handler needs to know the status of the various reset domains in the system so that the ping mechanism does not produce false alert indications.

Reviewer: ```suggestion
The alert handler needs to know the status of the various reset domains in the system to avoid false alert indications due to the ping mechanism.
```

Author: done

> To that end, the reset manager outputs a 4bit MuBi signal for each reset domain that indicates whether it is in reset or not.

Reviewer: ```suggestion
To that end, the reset manager outputs a 4bit MuBi signal for each reset domain that indicates whether its reset is active.
```

Or perhaps we can be more specific and say whether it can respond to ping requests.

Author: done
